mpi-proxy-split/lower-half/mem_wrapper.cpp

```diff
@@ -329,58 +329,3 @@ void updateMmaps(void *addr, size_t length) {
     }
   }
 }
```

Comment:
QUESTION:  I assume that we don't need this now because either DMTCP or MANA adds a page just beyond `sbrk(0)`, so that extending the heap is no longer possible.  Is that true?

If that's true, could you add a short comment here (in `mem_wrapper.cpp`) saying that:
> // We don't need a wrapper for sbrk() because [DMTCP? MANA?] has created a page above here to make sbrk(0) fail in libc/application.